README.md

```diff
@@ -50,33 +50,44 @@ Task.ordered {
 
 ### NSXPCConnection
 
-You might be tempted to make your XPC interface functions `async`. While this does work, it does not correctly handle connection failures and is unsafe. This little `NSXPCConnection` extension provides a safe way to get into the async world.
+You might be tempted to make your XPC interface functions `async`. This does not handle connection failures and can hang your `Task`s because it violates the Structured Concurrency contract, so it is unsafe. [See the post "ExtensionKit and XPC" for context.](https://www.chimehq.com/blog/extensionkit-xpc)
```

Comment:
What do you think about scoping the link just to the "ExtensionKit and XPC" part?